src/foundry/client/pixi/groups/primary.d.ts

```diff
@@ -1,40 +1,177 @@
-/**
- * A cached container group which renders the primary visible contents of a Scene.
- */
-declare class PrimaryCanvasGroup extends CachedContainer {
-  constructor();
+export {};
 
-  background: BackgroundLayer;
+type PrimaryDrawingContainer = unknown;
+type TokenMesh = unknown;
+type TileMesh = unknown;
+type TileSprite = unknown;
+type DrawingShape = unknown;
+type PrimaryCanvasObject = unknown;
```

Comment:
Another explicit TODO here might be good.

Reply:
All of these are also resolved on the PIXI-Layers branch